pipeline/src/resources/Client.py

```diff
@@ -0,0 +1,16 @@
+
+import socket
+
+# host = socket.gethostname()
+host = '127.0.0.1'
+port = 12345                   # The same port as used by the server
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+print('Connecting')
+s.connect((host, port))
+print('Connected')
+s.sendall(b'{"Hello": "world"}')
+print('Sent')
+data = s.recv(1024)
+print('Received')
+s.close()
+print('Received', repr(data))
```

Comment:
Is this whole file for testing purposes? This shouldn't be in the final pull request.

Reply:
I agree. I didn't want to make anyone looking at my code rewrite socket communication if they wanted to see if my code worked at all